src/main/java/org/gridsuite/modification/modifications/TabularCreation.java

```diff
@@ -57,6 +57,7 @@ public void apply(Network network, ReportNode subReportNode) {
         }
         String defaultMessage = switch (creationInfos.getCreationType()) {
             case GENERATOR_CREATION -> creationInfos.getCreations().size() > 1 ? "generators" : "generator";
+            case LOAD_CREATION -> creationInfos.getCreations().size() > 1 ? "loads" : "load";
```

Comment:
load is singular but the verbe is still plural ;-)
_Tabular creation: 1 load have been created_